### PR TITLE
Update Tray Icons Style

### PR DESCRIPTION
**Pull request by axionl**
## Preview
image

image

### CODE REVIEW

**Review by DuckSoft — CHANGES_REQUESTED** [review]
It seems that you've deleted `zh_CN.ts`. Was that intentional?

**Comment by axionl** [comment]
> It seems that you've deleted `zh_CN.ts`. Was that intentional?

I accidentally deleted this file, which thought it was local. Re-added it.

File: resources.new.qrc [hunk 1]
```diff
@@ -54,5 +54,20 @@
         <file>assets/icons/ui_light/tray.png</file>
         <file>assets/icons/ui_dark/netspeed_arrow.svg</file>
         <file>assets/icons/ui_light/netspeed_arrow.svg</file>
+        <file>assets/icons/ui_light/glyph-tray.svg</file>
+        <file>assets/icons/ui_light/glyph-tray-systemproxy.svg</file>
+        <file>assets/icons/ui_light/glyph-tray-connected.svg</file>
+        <file>../../../Desktop/ui_light/glyph-tray.png</file>
```
Comment (DuckSoft) [inline resources.new.qrc:60]: are these normal?

Comment (axionl) [reply resources.new.qrc]: Nope (x Orz...

**Review by DuckSoft — APPROVED** [review]
I am okay with the current pull request now, except a few harmless code format changes.
Let's turn it over to @U-v-U for the final decision.

**Review by U-v-U — APPROVED** [review]

**Pull request merged**